cfme/infrastructure/virtual_machines.py

```diff
@@ -708,6 +704,7 @@ def refresh(self):
     TO_RETIRE = "Retire this VM"
     VM_TYPE = "Virtual Machine"
     DETAILS_VIEW_CLASS = InfraVmDetailsView
+    SNAPSHOT_TITLE = 'name'
```

Comment:
Does this need to be here? I see you using SNAPSHOT_TITLE on the provider instance everywhere